Assets/Scripts/Actors/ActorComponents/PlayerActorInventory.cs

```diff
@@ -24,8 +24,11 @@ public List<BuddyTag> buddies
 	int resourceIndex = 0;
 	GameObject heldResource;
 
-	[SerializeField] LayerMask buddyLayer = 0;
-	[SerializeField] float maxGiveDistance = 2f;
+	[SerializeField] LayerMask _buddyLayer = 0;
+	public LayerMask buddyLayer
```

Comment:
Yeah, it's readonly to scripts, but can still be set from the inspector. This is a pattern that we should be using fairly regularly.